modules/microshift-adding-olm-to-blueprint.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-update-rpms-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-olm-to-blueprint_{context}"]
+== Adding the Operator Lifecycle Manager (OLM) service to a blueprint 
+
+When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can add the `microshift-olm` package in the ostree blueprint to enable OLM in {microshift-short}
+
+. Edit your ostree blueprint by running the following example command:
++
+[source,terminal]
+----
+$ vi minimal-microshift.toml
```

Comment:
```suggestion
$ vi <minimal-microshift>.toml <1>
```
<1> Replace <minimal-microshift> with the name of the blueprint file you used to add the MicroShift service.

I'd note here to replace <m-m> with the name of the blueprint because we're adding another blueprint they can use to the module before this one. If they used one of those in the preceding module, it'll be a different name.

Reply:
Gotcha, yeah that makes sense. I thought that was the name of the main one, like `install.config` in OCP. Updating!